src/components/FilmTime/FilmTimeHour.js

```diff
@@ -0,0 +1,21 @@
+const FilmTimeHour = [
+  {
+    id: 1,
+    time: ["12:30"],
```

Comment:
czemu to jest tablica?

Reply:
Zamist Stringa datę możesz przechowywać w klasie Date
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date